qrexec/tests/policy_parser.py

```diff
@@ -120,6 +134,11 @@ def test_010_Source(self):
         parser.Source("*")
         with self.assertRaises(exc.PolicySyntaxError):
             parser.Source("@default")
+        parser.Source("uuid:d8a249f1-b02b-4944-a9e5-437def2fbe2c")
```

Comment:
Please add a test with uppercase letters that don't raise a syntax error. [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122.html#section-3) allows case insensitive characters on input:

>    Each field is treated as an integer and has its value printed as a
>    zero-filled hexadecimal digit string with the most significant
>    digit first.  The hexadecimal values "a" through "f" are output as
>    lower case characters and are case insensitive on input.

Meaning it is syntactically correct for the user or program to write the uuid value using uppercase letters, but the Qrexec parser should evaluate them with case insensitivity and always output it in lower case.

Reply:
I agree on having a strict syntax check for lower case letters.